data.py

```diff
@@ -58,6 +58,7 @@
                 "Toolset",      # Various stand-alone applications for a given task
                 "Simulation",   # Generic code that can be used to run simulations described in the paper
                 "Experiments",  # Code that produces figures and graphs of the paper
+                "Hands-on",     # Stand alone hands-on workshop
```

Comment:
https://dictionary.cambridge.org/dictionary/english/stand-alone

```suggestion
                "Hands-on",     # Stand-alone hands-on workshop
```